doc/KubernetesClientWithIPv6Clusters.md

```diff
@@ -0,0 +1,40 @@
+# Using KubernetesClient with IPv6 based Kubernetes Clusters
+
+Right now Fabric8 Kubernetes Client doesn't work with IPv6 based Kubernetes Clusters due to an issue in OkHttp[square/okhttp#5889](https://github.com/square/okhttp/pull/5889). Fabric8 Kubernetes Client is right now dependent on OkHttp 3.x; unfortunately we can't upgrade to OkHttp 4.x due to it being now based on Kotlin, see [square/okhttp#4723](https://github.com/square/okhttp/issues/4723). We have decided not to upgrade to OkHttp 4.x. Until we find an alternative HTTP library for KubernetesClient, we suggest you to resolve this issue by excluding okhttp dependency coming from KubernetesClient jar and adding your own direct 4.x OkHttp dependencies:
```

Comment:
```suggestion
Right now Fabric8 Kubernetes Client doesn't work with IPv6 based Kubernetes Clusters due to an issue in OkHttp [square/okhttp#5889](https://github.com/square/okhttp/pull/5889). The issue is solved for OkHttp 4, but Fabric8 Kubernetes Client depends on OkHttp 3.x. ; unfortunately we can't upgrade to OkHttp 4.x due to it being now based on Kotlin, see [square/okhttp#4723](https://github.com/square/okhttp/issues/4723).

We have decided not to upgrade to OkHttp 4.x, because it's based on Kotilin ([square/okhttp#4723](https://github.com/square/okhttp/issues/4723)) and this might be an issue for downstream and dependent projects .

We're still discussing a proper replacement for OkHttp (#2764, #2632). In the meantime you can switch to use OkHttp 4.x in your project by configuring 3.x exclusions and adding your own direct 4.x OkHttp dependencies ([since it's binary compatible](https://github.com/fabric8io/kubernetes-client/issues/2632#issuecomment-748434878)):
```